src/cpp/include/openvino/genai/visual_language/pipeline.hpp

```diff
@@ -29,7 +30,22 @@ class OPENVINO_GENAI_EXPORTS VLMPipeline {
         const ov::AnyMap& properties = {}
     );
 
-    /// @brief Construct a pipeline form a folder containing tokenizer
+    /// @brief Construct a pipeline from a map of models.
+    /// @param models_map A map of models IRs (e.g. openvino_text_embeddings_model, openvino_language_model etc.).
+    /// @param tokenizer A tokenizer.
+    /// @param config_dir_path A path to directory containing config.json.
```

Comment:
```suggestion
    /// @param config_dir_path A path to directory containing preprocessor_config.json.
```

Reply:
For the pipeline we need `config.json`. `preprocessor_config.json` is needed in vision_encoder